plugins/node/opentelemetry-instrumentation-koa/README.md

```diff
@@ -54,6 +54,7 @@ Note that generator-based middleware are deprecated and won't be instrumented.
 | Options            | Type                                | Example              | Description                                                                                              |
 | ------------------ | ----------------------------------- | -------------------- | -------------------------------------------------------------------------------------------------------- |
 | `ignoreLayersType` | `KoaLayerType[]`                    | `['middleware']`     | Ignore layers of specified type.                                                                         |
+| `ignoreLayers`     | `IgnoreMatcher[]`                   | `['logger', /router/]`         | Ignore layers with specified names.                                                                      |
 | `requestHook`      | `KoaRequestCustomAttributeFunction` | `(span, info) => {}` | Function for adding custom attributes to Koa middleware layers. Receives params: `Span, KoaRequestInfo`. |
 
 `ignoreLayersType` accepts an array of `KoaLayerType` which can take the following string values:
```

Comment:
Please also include the notes about `ignoreLayers` from the express README: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/5eb61d8ea51371bea1b6030dca9c78861f4cd6d4/plugins/node/opentelemetry-instrumentation-express/README.md#L65-L69

Reply:
Missed that, thanks!